### PR TITLE
[GPT-728] Full compilation via Template Unified Compiler

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

File: src/cli/domain/package-components.js [hunk 1]
```diff
@@ -2,131 +2,57 @@
...
-            return cb(null, component);
-          }
-
-          packageServerScript(
-            {
-              componentPath: componentPath,
-              dependencies: component.dependencies,
-              ocOptions: component.oc,
-              publishPath: publishPath,
-              verbose: options.verbose
```
Comment (matteofigus) [inline src/cli/domain/package-components.js]: we have a verbose here...

File: src/cli/domain/package-components.js [hunk 2]
```diff
@@ -2,131 +2,57 @@
...
-            err => cb(err, component)
-          );
+    const type = componentPackage.oc.files.template.type;
+    const compileOptions = {
+      publishPath,
+      componentPath,
+      componentPackage,
+      ocPackage,
+      minify
+      // TODO: verbose,
```
Comment (matteofigus) [inline src/cli/domain/package-components.js]: ...so, should we start passing it through, even if the current compilers will ignore it?

Comment (nickbalestra) [inline src/cli/domain/package-components.js]: I would say yes

**Comment by nickbalestra** [comment]
Passed `verbose` to it. Thanks for the suggestion.

**Pull request merged**